exporter.py

```diff
@@ -129,9 +129,13 @@ def _report_log(cls, logs):
                     print("      ", log.human_duration, ":", log.comment)
 
     @classmethod
-    def report(cls, to_create, to_delete, to_check, attendances=None):
+    def report(cls, to_create, to_delete, to_check, to_dispatch=[], attendances=None):
```

Comment:
never use non hashable default values as arguments

Reply:
To be changed of course ^^